clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -10226,7 +10229,7 @@ Value *CodeGenFunction::EmitSVETupleCreate(const SVETypeFlags &TypeFlags,
                                              ArrayRef<Value *> Ops) {
   assert(TypeFlags.isTupleCreate() && "Expects TypleFlag isTupleCreate");
 
-  auto *SrcTy = dyn_cast<llvm::ScalableVectorType>(Ops[0]->getType());
+  auto *SrcTy = cast<llvm::ScalableVectorType>(Ops[0]->getType());
```

Comment:
I believe we should have some consistency in here between EmitSVETupleSetOrGet and EmitSVETupleCreate.
Or both we do :
 auto *SrcTy/SingleVecTy) = cast<llvm::ScalableVectorType>
or 
auto *SrcTy/SingleVecTy) = dyn_cast<llvm::ScalableVectorType>(
 if (!ScrTy/SingleVecTy))
   return 

Maybe is fine to cast directly to ScalableVectorType. I can see this being done is the other EmitSVE.

Reply:
Thanks @CarolineConcatto for reviews. I have updated patch to be consistent with SVETuple functions.